src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx

```diff
@@ -99,60 +113,134 @@ const UploadFileUI = ({
   const showCreateFolderModal = () => dispatch(showFolderModal());
   const hideCreateFolderModal = () => dispatch(hideFolderModal());
   const dragBoxShow = () => dispatch(showDrageBox());
-  const dagBoxHide = () => dispatch(hideDrageBox());
+  const dragBoxHide = () => dispatch(hideDrageBox());
   const uploadOptionsHide = () => dispatch(hideUploadOptions());
   const uploadOptionsShow = () => dispatch(showUploadOptions());
   const hideButtonAdd = () => dispatch(hideAddButton());
   const shownAddButton = () => dispatch(showAddButton());
   const StatushideFile = () => dispatch(hideFileStatus());
   const crossBtnhide = () => dispatch(hideCrossBtn());
   const showFiles = () => dispatch(showUploadFiles());
+  const shownFolder = () => dispatch(showFolder());
+
   const router = useRouter();
   const campRoute = () => {
     router.push("/create-new-topic");
   };
   const validateMessages = {
     required: "${name} is required !",
   };
-  const menu = (i) => (
+  const menu = (i, obj) => (
     <Menu>
       <Menu.Item>
-        <a onClick={() => Openfolder(i)}>Open folder</a>
+        <a
+          onClick={() => {
+            Openfolder(i);
+            setFolderIndex(i);
+          }}
+        >
+          Open folder
+        </a>
       </Menu.Item>
       <Menu.Item>
-        <a>Edit folder</a>
+        <a onClick={() => editFolder(obj)}>Edit folder</a>
       </Menu.Item>
       <Menu.Item>
-        <a>Delete folder</a>
+        <a
+          onClick={() => {
+            removeFiles(obj, {}, fileLists);
+          }}
+        >
+          Delete folder
+        </a>
       </Menu.Item>
     </Menu>
   );
-  const menu_files = (i) => (
+  const menu_files = (i, item) => (
     <Menu>
       <Menu.Item>
-        <a>
+        <a
+          className={styles.high_light}
+          onClick={() =>
+            setPreview({
+              previewVisible: true,
+              previewName: item.name,
+              previewPath: item.thumbUrl,
+            })
+          }
+        >
           <EyeTwoTone /> View File
         </a>
       </Menu.Item>
       <Menu.Item>
-        <a>
+        <a
```

Comment:
Please use `Link` on all the anchor links

Reply:
changed to div